Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -8,16 +8,13 @@
 import SwiftUI
 
 struct SettingsView: View {
-    
-    @StateObject private var viewModel: SettingsViewModel
+    private var viewModel: SettingsViewModel
```

Comment:
Does this need to be `@Bindable`

Reply:
Same as above:
here we do not use $viewModel.xxxxx
So according to the docs, this should not be @Bindable